core/bootstrap/src/main/java/org/jboss/modules/maven/MavenArtifactUtil.java

```diff
@@ -59,6 +62,8 @@
 
     private static XPathExpression snapshotVersionXpath;
 
+    private static final Pattern pattern = Pattern.compile("\\S+[0-9]{5,}.\\S{5,}");
```

Comment:
What does this mean? Quite non-descriptive variable name :-)

Reply:
LOL you're right sorry. Fixing now